cpu/lm4f120/periph/uart.c

```diff
@@ -138,12 +129,13 @@ void isr_uart0(void)
     ROM_UARTIntClear(UART0_BASE, ulStatus);
 
     /* Are we interrupted due to a received character */
-    if(ulStatus & (UART_INT_RX | UART_INT_RT))
+    if (ulStatus & (UART_INT_RX | UART_INT_RT))
     {
-        while(ROM_UARTCharsAvail(UART0_BASE))
+        while (ROM_UARTCharsAvail(UART0_BASE))
         {
```

Comment:
```suggestion
if (ulStatus & (UART_INT_RX | UART_INT_RT)) {
    while (ROM_UARTCharsAvail(UART0_BASE)) {
```
(doesn't render well sorry)